WasatchNET/Spectrometer.cs

```diff
@@ -211,6 +270,22 @@ public virtual double[] dark
         /// </summary>
         // private bool kludgedOut = false;
 
+        public uint? acquisitionTimeoutMS { get; set; }

```

Comment:
Action: document

Reply:
Thank you: other two make sense to me, this one less so